packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -23,6 +23,7 @@ import { css, registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixi
 const multiSelectComboBox = css`
   :host {
     --input-min-width: var(--vaadin-multi-select-combo-box-input-min-width, 4em);
+    --chip-min-width: var(--vaadin-multi-select-combo-box-chip-min-width, 50px);
```

Comment:
Should this be a private CSS variable (`--_chip-min-width`)?

Reply:
Renamed.